cmd/cluster/clusterCreate.go

```diff
@@ -331,6 +331,7 @@ func NewCmdClusterCreate() *cobra.Command {
 
 	cmd.Flags().String("registry-config", "", "Specify path to an extra registries.yaml file")
 	_ = cfgViper.BindPFlag("registries.config", cmd.Flags().Lookup("registry-config"))
+	_ = cfgViper.BindEnv("registries.config")
```

Comment:
Why do we need to do this? Shouldn't the `cfgViper.AutomaticEnv` ensure we can use flags or envs?

Reply:
you are right! we only need the call to SetEnvKeyReplacer to access struct members